admin/src/components/AdminAvatar/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { AdminAvatarWrapper, AdminShield } from "./styles";
+// @ts-ignore
+import {Shield} from "@strapi/icons";
+// @ts-ignore
+import {Icon} from "@strapi/design-system/Icon";
+import { ToBeFixed } from "../../../../types";
+
+const AdminAvatar = ({ children }: ToBeFixed ) => {
```

Comment:
We know the type of children. How about this solution? 
```suggestion
const AdminAvatar:  React.FC<React.PropsWithChildren<{}>> = ({ children }) => {
```